source/mainnet/net/installation/downloads.rst

```diff
@@ -14,14 +14,17 @@ This topic contains information about where you can download the Concordium Wall
 |mw-gen2|
 ========================
 
-The |mw-gen2| is available for Android™. The |mw-gen2| supports Android 8 or later.
+The |mw-gen2| is available for iOS and Android™. The |mw-gen2| supports iOS 13 or later and Android 8 or later.
 
 .. Note::
 
    The |mw-gen2| is not supported on tablet devices.
 
-You can find |mw-gen2| on Google Play.
+You can find |mw-gen2| on App Store and on Google Play.
 
+   .. image:: ../images/mobile-wallet/app-store-badge.svg
+      :width: 23%
+      :target: https://apps.apple.com/us/app/concordium-mobile-wallet/id1566996491
```

Comment:
```suggestion
      :target: https://apps.apple.com/us/app/concordium-blockchain-wallet/id6444703764
```

Reply:
@dg-concordium @concordium-cl This should be the link to the new wallet right? I just to want to double check that my correction is correct before I add commit the suggestion

Reply:
I guess it is but we can test it before it's released.